duckduckgo/macos-browser
# Fix error page being shown after a download

Task/Issue URL: https://app.asana.com/0/1201011656765697/1203133928050237/f
Tech Design URL:
CC:

**Description**:
Fix issue where the FrameInterrupted error was not being correctly handled during downloads.

**Steps to test this PR**:
1. Go to a site to trigger a download, for example [this](https://discadia.com/emojis/?q=panda) or [this](https://www.thinkbroadband.com/download) 
2. Check if the download starts correctly and no error page is shown
3. Test general error handling, for example:  Trying to open a page that doesn't exists or a page where it's blocked by your network (I've tested this blocking some domains on my DNS server). Check if we correctly display the error page

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
